sw/device/tests/dif/dif_entropy_smoketest.c

```diff
@@ -45,8 +45,6 @@ bool test_main() {
       .reset_health_test_registers = false,
       .single_bit_mode = kDifEntropySingleBitModeDisabled,
       .route_to_firmware = true,
-      .sample_rate = 2,
-      .lfsr_seed = 2,
   };
```

Comment:
Same comment as with the unit test, less is better, even though i keep the .mode field.